tlv493d/tlv493d.go

```diff
@@ -422,6 +425,18 @@ func (d *Dev) StopContinousRead() {
 	d.continuousReadWG.Wait()
 }
 
+// CalibrateTemperatureOffsetCompensation computes the temperature offset compensation based upon a reading from the sensor compared to the actual temperature.
+// The returned value must be stored and passed in Opts.TemperatureOffsetCompensation in future uses of TLV493D driver.
```

Comment:
The doc style is to keep an empty line after the describing sentence. Not a big deal.

Reply:
Fixed